src/setup.py

```diff
@@ -41,7 +41,8 @@ def find_version(*file_paths):
     'pmr2.client >= 0.2',
     'packaging',
     'filelock',
-    'psutil == 5.9.3'
+    'psutil == 5.9.3',
+    'PyGithub'
```

Comment:
I can't see the reason for this addition, is it used somewhere?

Reply:
Yes, it is used by the Plugin Finder tool to make calls to the GitHub API, which is how we interact with our database. The import statements are slightly misleading because the PyGithub package is imported with `from github import Github`. The import statement `from github.GithubException import BadCredentialsException, RateLimitExceededException` is also referring to PyGithub.